src/EventListener/AccessDeniedExceptionEventSubscriber.php

```diff
@@ -32,7 +32,7 @@ public static function getSubscribedEvents(): array
     public function onException(ExceptionEvent $event): void
     {
         $exception = $event->getThrowable();
-        if (!$exception instanceof AccessDeniedHttpException) {
+        if (!$exception instanceof UnauthorizedHttpException) {
```

Comment:
As this class now catches `UnauthorizedHttpException`, we should rename: 
- the class itself in `UnauthorizedExceptionEventSubscriber`
- the warning log message should be `An "Unauthorized" error was detected, destroying the session.`
- the unit test